extensions/wikia/AdEngine/js/slot/inContent.js

```diff
@@ -2,11 +2,12 @@
 define('ext.wikia.adEngine.slot.inContent', [
 	'ext.wikia.adEngine.adTracker',
 	'ext.wikia.adEngine.context.slotsContext',
+	'ext.wikia.adEngine.video.videoFrequencyMonitor',
 	'JSMessages',
 	'wikia.document',
 	'wikia.log',
 	'wikia.window'
-], function (adTracker, slotsContext, msg, doc, log, win) {
+], function (adTracker, slotsContext, videoFrequencyMonitor, msg, doc, log, win) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has 7 parameters which is greater than 4 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)